src/NuGet.Services.AzureSearch/Models/CommittedDocument.cs

```diff
@@ -24,6 +25,7 @@ public abstract class CommittedDocument : KeyedDocument, ICommittedDocument
         public bool? LastUpdatedFromCatalog { get; set; }
 
         [JsonProperty(NullValueHandling = NullValueHandling.Include)]
+        [IsSortable]
```

Comment:
Super nit: What do you think of putting `IsSortable` before `JsonProperty`? I think it looks a little nicer to have the shorter attribute first.

Reply:
Sure, seems fine.